app/src/main/java/com/github/epfl/meili/models/Review.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.epfl.meili.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Review (
+    var rating: Float = 0f,
```

Comment:
Since this is a constructor, by putting default values for arguments I'm allowing Firestore to call the empty constructor for Review. This is a requirement for Firestore if we want to use the toObject function of a documentSnapshot, which is used in FirestoreReviewService.